spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/catalog/TestTransformTableChange.java

```diff
@@ -160,4 +160,46 @@ void testTransformUpdateColumnType() {
     Assertions.assertTrue(
         "string".equalsIgnoreCase(gravitinoUpdateColumnType.getNewDataType().simpleString()));
   }
+
+  @Test
+  void testTransformUpdateColumnPosition() {
+    TableChange.ColumnPosition first = TableChange.ColumnPosition.first();
+    TableChange.ColumnPosition after = TableChange.ColumnPosition.after("col0");
+
+    TableChange.UpdateColumnPosition sparkUpdateColumnFirst =
+        (TableChange.UpdateColumnPosition)
+            TableChange.updateColumnPosition(new String[] {"col1"}, first);
+    com.datastrato.gravitino.rel.TableChange gravitinoChangeFirst =
+        GravitinoCatalog.transformTableChange(sparkUpdateColumnFirst);
+
+    Assertions.assertTrue(
+        gravitinoChangeFirst
+            instanceof com.datastrato.gravitino.rel.TableChange.UpdateColumnPosition);
+    com.datastrato.gravitino.rel.TableChange.UpdateColumnPosition gravitinoUpdateColumnFirst =
+        (com.datastrato.gravitino.rel.TableChange.UpdateColumnPosition) gravitinoChangeFirst;
+
+    Assertions.assertEquals(
```

Comment:
could you replace `assertEquals` with `assertArrayEquals` which compares the array content.